test/webapi/test_context.py

```diff
@@ -221,3 +221,246 @@ def test_interpolates_vars(self):
                          normalize_prefix('/${name}'))
         self.assertEqual(f'/xcube/v{version}',
                          normalize_prefix('/${name}/v${version}'))
+
+
+class MaybeAssignStoreInstanceIdsTest(unittest.TestCase):
+
+    def test_find_common_store(self):
+        ctx = new_test_service_context()
+        dataset_configs = [
+            {
+                'Identifier': 'z_0',
+                'FileSystem': 'local',
+                'Path': '/one/path/abc.zarr'
+            },
+            {
+                'Identifier': 'z_1',
+                'FileSystem': 'local',
+                'Path': '/one/path/def.zarr'
+            },
+            {
+                'Identifier': 'z_4',
+                'FileSystem': 'obs',
```

Comment:
I think we should stop using "obs" when referring to s3 object storage - and exchange it to "s3" as used in the DataStores configurations.

Reply:
Agreed!

Reply:
Correct - but for now, we'll move this to another PR.